index.ios.js

```diff
@@ -1,30 +1,5 @@
-/**
- * Fake class.
- * isHardwareDetected() should be enough on it's own, which is the reason not everything has been stubbed.
- * (and if you are actually smart enough to call authenticate() it will just crash...)
- */
 
-export default class FingerprintAndroid {
-    static async cancelAuthentication(): Promise<null> { 
-        return null;
-    };
-    static async isAuthenticationCanceled(): Promise<boolean> { 
-        return false;
-    };
-
-    static async hasPermission(): Promise<boolean> {
-        return false;
-    };
-
-    static async hasEnrolledFingerprints(): Promise<boolean> {
-        return false;
-    };
-
-    static async isHardwareDetected(): Promise<boolean> {
-        return false;
-    };
-
-    static async authenticate(warningCallback:?(response:FingerprintError) => {}):Promise<null> {
-        throw new Error("Believe it or not, but react-native-fingerprint-android actually doesn't support iOS.")
-    }
-}
\ No newline at end of file
+throw new Error(
```

Comment:
Why throw an error on iOS _on import_? That's really annoying for users of the lib!  
The whole reason I made it a fake module is so that you can import it on both platforms without it interfering.  
Conditional imports suck. There's no reason to immediately crash.